src/js/pinny.js

```diff
@@ -148,7 +156,15 @@
                         $window.off(events.orientationchange);
                     }
 
-                    this.$container.attr('aria-hidden', 'false');
+                    // Lockup a11y fix
+                    // Instead of making whole lockup container aria hidden,
+                    // just make page content aria hidden, since otherwise pinny
+                    // cannot be read by screenreaders
+                    if ($('#x-root').length) {
+                        $('#x-root').attr('aria-hidden', 'false');
```

Comment:
Good point @jeffkamo . I'll set it as an overridable option specified during init.